src/sentry/models/sentryfunction.py

```diff
@@ -13,7 +14,7 @@ class SentryFunction(DefaultFieldsModel):
     external_id = models.CharField(max_length=128, unique=True)
     overview = models.TextField(null=True)
     code = models.TextField(null=True)
-
+    events = ArrayField(of=models.TextField, null=True)
```

Comment:
If you want to have a default of `[]` just in code you can set that here and use the state migration you had before to have the default controlled by django code rather than the db.